src/thermo/PlasmaPhase.cpp

```diff
@@ -0,0 +1,272 @@
+//! @file PlasmaPhase.cpp
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#include "cantera/thermo/PlasmaPhase.h"
+#include "cantera/thermo/Species.h"
+#include "cantera/base/global.h"
+#include "cantera/numerics/funcs.h"
+#include <boost/math/special_functions/gamma.hpp>
+
+namespace Cantera {
+
+PlasmaPhase::PlasmaPhase(const std::string& inputFile,
+                         const std::string& id_)
+    : m_isotropicShapeFactor(2.0)
+    , m_nPoints(1000)
```

Comment:
I guess given the implementation requirements of Simpson's rule, it would be better if this defaulted to an odd number, right?